cluster/sync-olm.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash -x
+
+# sync-olm syncs local code changes to the k8s/OCP cluster with OLM enabled on it.
+# It aims to be the single command that a dev needs to run to sync changes
+# instead of manually running a combination of below commands:
+# - make build-operator
+# - make build-webhook
+# - make build-manifests
+# - make container-build
+# - make container-push
+# - Replacing image names in couple of files
+# - make bundleRegistry
+# - ./hack/build-index-image.sh
+#
+# Prerequisites:
+# - Bunch of variables to build container images and push them to a container registry.
+# - Access to a k8s/OCP cluster that has OLM enabled on it.
+
+# tag for the bundle image
+CONTAINER_TAG="${CONTAINER_TAG:-$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 8 | head -n 1)}"
+IMAGE_REGISTRY="${IMAGE_REGISTRY:-quay.io}"
+# image to be replaced in CSV yaml files
+IMAGE_TO_REPLACE=$IMAGE_REGISTRY/$REGISTRY_NAMESPACE/hyperconverged-cluster-operator:$CONTAINER_TAG
```

Comment:
this is unused

Reply:
did you forget to add the steps as you mentioned above in:

> Modify all the occurrences of `+IMAGE_TO_REPLACE+` in `deploy/index-image/community-kubevirt-hyperconverged/1.10.0/manifests/kubevirt-hyperconverged-operator.v1.10.0.clusterserviceversion.yaml`
> ...

Reply:
It's actually being taken care of by:
```sh
# Image to be used in CSV manifests
HCO_OPERATOR_IMAGE=$IMAGE_REGISTRY/$REGISTRY_NAMESPACE/hyperconverged-cluster-operator
HCO_OPERATOR_IMAGE=$HCO_OPERATOR_IMAGE CSV_VERSION=$CSV_VERSION make build-manifests
```
So the variable should be removed. Thanks for the catch. 👍🏾

Reply:
Then why do we say to:

> Next, modify the occurrences of `+IMAGE_TO_REPLACE+` in `deploy/index-image/community-kubevirt-hyperconverged/1.10.
> 0/manifests/kubevirt-hyperconverged-operator.v1.10.0.clusterserviceversion.yaml` with the container image created by
> `make container-build` command. 
> ....

wouldn't it be easier to say to execute those command too?